ts/src/base/Exchange.ts

```diff
@@ -3765,12 +3765,19 @@ export default class Exchange {
             precision = this.safeValue (networkItem, 'precision', precision);
         }
         if (precision === undefined) {
-            return fee;
+            return this.forceString (fee);
         } else {
             return this.decimalToPrecision (fee, ROUND, precision, this.precisionMode, this.paddingMode);
         }
     }
 
+    forceString (value) {
```

Comment:
@ttodua why do we need this method? `numbertoString` already handles both cases (string inputs and number inputs)

Reply:
as mentioned in the past, `numberToString` is slightly different, please take a look: https://github.com/ccxt/ccxt/blob/3481addf7d641081eb2f7763e54c20086faf8935/ts/src/base/functions/number.ts#L40
even if you provide `string`, it re-parses it and does extra things. but `forceString` just returns back if `string` was passed.